modules/gatsby/src/components/layout/table-of-contents.jsx

```diff
@@ -47,7 +47,15 @@ export default class TableOfContents extends PureComponent {
     this.toggleEntry = this.toggleEntry.bind(this);
   }
 
+
+  componentDidMount() {
+    console.log('CDM');
```

Comment:
Drop? 

Or import probe.gl so that we can enable ocular debug logging conditionally.

Right now there are lots of warnings in ocular console that are quite distracting...

Reply:
thanks